docs/manual/java/guide/build/EclipseSbt.md

```diff
@@ -0,0 +1,57 @@
+# Importing an sbt project into Eclipse
+
+If you worked through the command line example for [[sbt|GettingStartedSbt]], you have an sbt build. The sbt tool provides an [sbt-eclipse](https://github.com/typesafehub/sbteclipse) plugin that generates Eclipse project artifacts, which you can then import into Eclipse.
+
+Follow these steps to integrate your project with Eclipse:
+
+1. Configure sbt to include the Eclipse plugin:
+    
+    1. In the `project` folder of your sbt build, create an `eclipse.sbt` file. For example, with a project named `hello`, create a `hello/project/eclipse.sbt` file. 
```

Comment:
The version number goes both ways - if we refer to a specific version number that we know works, that's good - sbteclipse can (and likely will) introduce regressions that break things for us, so having a version here can help.